tests/functional/pages/landing.page.js

```diff
@@ -10,6 +10,26 @@ class LandingPage extends BasePage {
     return '.qa-btn-get-started';
   }
 
+  useOfInformation() {
+    return '#use-of-information';
+  }
+
+  useOfData() {
+    return '#how-we-use-your-data';
+  }
+
+  legalResponse() {
+      return '#legal-response';
```

Comment:
I don't think this 'id' exists on the introduction page. Might be worth adding 'id's of 'data-qa's to the html to make testing easier.